R/load_weights.R

```diff
@@ -0,0 +1,157 @@
+
+# make_and_load_bert ------------------------------------------------------
+
+
+#' Pretrained BERT Model
+#'
+#' Construct a BERT model and load pretrained weights.
+#'
+#' @param model_name Character; which flavor of BERT to use.
```

Comment:
Add an issue to present this information more clearly to the user (what the supported options are).